test_resume.py

```
@@ -0,0 +1,230 @@
+import io
+import json
```

Ruff (0.12.2)

`json` imported but unused

Remove unused import: `json`

(F401)